lib/system/cellsets.nim

```diff
@@ -169,6 +169,16 @@ proc excl(s: var CellSet, cell: PCell) =
     t.bits[u shr IntShift] = (t.bits[u shr IntShift] and
                               not (1 shl (u and IntMask)))
 
+proc missingOrExcl(s: var CellSet, cell: PCell): bool =
```

Comment:
This is not required, the GC doesn't need ``missingOrExcl``.

Reply:
Haha, I was thinking the same thing, almost removed it before. Sounds good to me